src/passes/AvoidReinterprets.cpp

```diff
@@ -20,11 +20,11 @@
 // beneficial, unless reinterprets are very costly (which is the case
 // with wasm2js).
 
-#include <ir/local-graph.h>
-#include <ir/properties.h>
-#include <pass.h>
-#include <wasm-builder.h>
-#include <wasm.h>
+#include "ir/local-graph.h"
+#include "ir/properties.h"
+#include "pass.h"
+#include "wasm-builder.h"
+#include "../wasm.h"
```

Comment:
I think changes from `<` to `"` definitely make sense to do.

But I hope we can avoid the `../` changes. For those, wouldn't it work if the binaryen sources are built with different `-I` paths than the wasmtime sources?

Reply:
If every path was made relative throughout the codebase yeah binaryen could just be included differently but it would be as `src/wasm.h` not `binaryen/wasm.h`

Reply:
Sorry, I don't follow. By "codebase" which codebase do you mean?

Reply:
In binaryen sorry, making every single path in binaryen `../[something]` and using no `<something.h>` would make that possible, but that would just create even more `../` changes. Binaryen can't be given its own personal include path so one project would have to refer to only relative paths. There might be another fix but I don't know of it, maybe with CMake.

Reply:
> Binaryen can't be given its own personal include path so one project would have to refer to only relative paths

There should be a way to do this in most build systems I hope. For example in CMake if you put Binaryen in one directory you could add include paths in the CMakeLists.txt in that directory which only take effect there.

Reply:
Are you saying I use binaryen and wasmtime in separate projects and just link one to the other? I definitely could do that, that would fix the include clashing. I can close this PR then

Reply:
Yes, not building their sources all at once and just linking the compiled code is a good way to avoid such issues. Different build systems help do that in different ways (I mentioned how CMake does it; I'm less familiar with others).